zebedee-cms/src/main/java/com/github/onsdigital/zebedee/util/slack/Notifier.java

```diff
@@ -17,12 +19,23 @@
      * @param args - additional arguments to add to the notification.
      */
     void collectionAlarm(Collection c, String alarm, PostMessageField... args);
-
     /**
      * Create a new alarm notification.
      *
      * @param alarm - the string message to apply to the notification.
      * @param args - additional arguments to add to the notification.
      */
     void alarm(String alarm, PostMessageField... args);
+
+    void sendSlackMessage(PostMessage message) throws Exception;
+
+    PostMessage createPostMessage(String channel, String text);
+
+    void callCollectionAlarm(Collection c, String channel, String message, Exception ex);
```

Comment:
I think we should rename these 3 methods - `sendCollectionAlarm`/ `sendCollectionWarning` sounds more appropriate to me.